doc/source/tokudb/tokudb_variables.rst

```diff
@@ -911,6 +921,29 @@ output:
 Defines direction to be used to perform table scan to check for empty tables
 for bulk loader.
 
+.. variable:: tokudb_enable_fast_update
+
+  :cli: Yes
+  :conf: Yes
+  :scope: Global/Session
+  :dyn: Yes
+  :vartype: Boolean
+  :default: Off
+
+Enables the fast updates feature, which substantially facilitates execution of
```

Comment:
Expand

Reply:
Details on how/why one might use this are here https://www.percona.com/blog/2013/02/12/fast-updates-with-tokudb/ but note that the actual syntax of invoking/enabling this functionality (keyword NOAR vs session variable) is different between this very old blog post and the implementation that is being documented here.